<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SR_editQtWidgets</class>
 <widget class="QWidget" name="SR_editQtWidgets">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGraphicsView" name="graphicsView"/>
   </item>
   <item>
    <widget class="QWidget" name="video_toole_widget2" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="in_timelabel">
        <property name="text">
         <string>inTime</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="MySlider" name="horizontalSlider">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>20</height>
         </size>
        </property>
        <property name="maximum">
         <number>1000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="out_timelabel">
        <property name="text">
         <string>outTime</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="video_toole_widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="JtoolButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../Resource.qrc">
            <normaloff>:/new/prefix1/Resource/image/winOCRUI/backward.svg</normaloff>:/new/prefix1/Resource/image/winOCRUI/backward.svg</iconset>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton_2">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../Resource.qrc">
            <normaloff>:/new/prefix1/Resource/image/winOCRUI/play.svg</normaloff>:/new/prefix1/Resource/image/winOCRUI/play.svg</iconset>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="LtoolButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../Resource.qrc">
            <normaloff>:/new/prefix1/Resource/image/winOCRUI/forward.svg</normaloff>:/new/prefix1/Resource/image/winOCRUI/forward.svg</iconset>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="setInTimetoolButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../Resource.qrc">
            <normaloff>:/new/prefix1/Resource/image/winOCRUI/arrow.right.to.line.svg</normaloff>:/new/prefix1/Resource/image/winOCRUI/arrow.right.to.line.svg</iconset>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="setOutTimetoolButton">
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../Resource.qrc">
            <normaloff>:/new/prefix1/Resource/image/winOCRUI/arrow.backward.to.line.svg</normaloff>:/new/prefix1/Resource/image/winOCRUI/arrow.backward.to.line.svg</iconset>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="nowtimelabel">
          <property name="text">
           <string>TextLabel</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>推理模型:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="model_Instrumental_selection_comboBox"/>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>模型放大倍数:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="scaled_label">
       <property name="text">
        <string>未知</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>输出格式:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="video_out_type_widget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QRadioButton" name="video_radioButton_default">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>跟随输入</string>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">out_video_buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="video_radioButton_mp4">
          <property name="text">
           <string>mp4</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">out_video_buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="video_radioButton_mov">
          <property name="text">
           <string>mov</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">out_video_buttonGroup</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="image_out_type_widget" native="true">
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="leftMargin">
         <number>2</number>
        </property>
        <property name="topMargin">
         <number>2</number>
        </property>
        <property name="rightMargin">
         <number>2</number>
        </property>
        <property name="bottomMargin">
         <number>2</number>
        </property>
        <item>
         <widget class="QRadioButton" name="image_radioButton_png">
          <property name="text">
           <string>PNG</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">out_image_buttonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QRadioButton" name="image_radioButton_jpg">
          <property name="text">
           <string>JPG</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">out_image_buttonGroup</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="zoom_in_pushButton">
       <property name="text">
        <string>+</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="zoom_out_pushButton">
       <property name="text">
        <string>-</string>
       </property>
       <property name="flat">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QPushButton" name="tohomepushButton">
     <property name="text">
      <string>保存返回列表</string>
     </property>
     <property name="icon">
      <iconset resource="../Resource.qrc">
       <normaloff>:/new/prefix1/Resource/image/winOCRUI/arrowshape.turn.up.left.fill.svg</normaloff>:/new/prefix1/Resource/image/winOCRUI/arrowshape.turn.up.left.fill.svg</iconset>
     </property>
     <property name="default">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MySlider</class>
   <extends>QSlider</extends>
   <header>myslider.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Resource.qrc"/>
 </resources>
 <connections/>
 <buttongroups>
  <buttongroup name="out_image_buttonGroup"/>
  <buttongroup name="out_video_buttonGroup"/>
 </buttongroups>
</ui>
